src/main/steps/c100-rebuild/miam/miam-info/template.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "common/template.njk" %}
+
+
```

Comment:
please check if we can add all these as part of content.ts

Reply:
These all fields are as part of content.ts. it's been just referred here